bigtable-client-core-parent/bigtable-hbase/pom.xml

```diff
@@ -203,6 +210,40 @@ limitations under the License.
           </systemProperties>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>com.google.cloud.bigtable.test</groupId>
+        <artifactId>bigtable-build-helper</artifactId>
+        <version>1.14.1-SNAPSHOT</version> <!-- {x-version-update:bigtable-test:current} -->
```

Comment:
can we use the project.version variable instead?

Reply:
its different artifacts build-helper uses bigtable-test version while project.version follows bigtable-client-parent version. While we have the split in versions.txt, we should keep this line as is. In the future we can collapse everything to a single identifier in versions.txt and then update this